src/app/admin/backup.ts

```diff
@@ -63,7 +63,8 @@ export const uploadBackup =
             addEventToCurrentSpan("Static channel backup to Dropbox successful.")
           } catch (error) {
             logger.error({ error }, "Static channel backup to Dropbox failed.")
-            recordExceptionInCurrentSpan({ error: error, level: ErrorLevel.Warn })
+            if (error instanceof Error)
```

Comment:
this is not great you are now potentially silently eating an error. We still need to be alerted of all errors.